iOS-Email-Client/Controllers/ProfileEditorViewController.swift

```diff
@@ -64,44 +64,38 @@ class ProfileEditorViewController: UIViewController {
     @IBOutlet weak var tableView: UITableView!
     @IBOutlet weak var attachmentController: AttachmentOptionsContainerView!
     @IBOutlet weak var attachmentContainerBottomConstraint: NSLayoutConstraint!
+    @IBOutlet weak var scrollContentViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var tableViewHeightConstraint: NSLayoutConstraint!
     let STATUS_NOT_CONFIRMED = 0
     let SECTION_PROFILE = 0
-    let ROW_HEIGHT: CGFloat = 40.0
+    let ROW_HEADER_HEIGHT: CGFloat = 40.0
+    let ROW_HEIGHT: CGFloat = 60.0
     var imagePicker = UIImagePickerController()
-    var attachmentOptionsHeight: CGFloat = 0
+    var attachmentOptionsBottomPadding: CGFloat = -120
```

Comment:
If this is a constant, should be with the same format as others.
`var ATTACHMENT_OPTIONS_BOTTOM_PADDING: CGFloat = -120`